cf/rds.yaml

```diff
@@ -35,6 +36,40 @@ Parameters:
     Default: aurora
 
 Resources:
+  RDSMasterUserSecret:
+    Type: AWS::SecretsManager::Secret
+    Properties:
+      Description: RDS master secret
+      Name: !Sub "${DBSecretPrefix}-${Environment}"
+      GenerateSecretString:
+        SecretStringTemplate: '{"username": "postgres"}'
+        GenerateStringKey: password
+        PasswordLength: 32
+        ExcludeCharacters: "/@\"'\\"
+
+  RDSInstanceAttachment:
+    Type: AWS::SecretsManager::SecretTargetAttachment
+    Properties:
+      SecretId: !Ref RDSMasterUserSecret
+      TargetId: !Ref DBInstance1
+      TargetType: AWS::RDS::DBInstance
+
+  RDSSecretRotationSchedule:
+    Type: AWS::SecretsManager::RotationSchedule
+    DependsOn: RDSInstanceAttachment
+    Properties:
+      SecretId: !Ref RDSMasterUserSecret
+      HostedRotationLambda:
+        RotationType: PostgreSQLSingleUser
+        RotationLambdaName: !Sub "RDSSecretsManagerRotation${Environment}"
+        VpcSubnetIds:
+          Fn::ImportValue: !Sub "eksctl-biomage-${Environment}-cluster::SubnetsPrivate"
+        VpcSecurityGroupIds:
+          Fn::ImportValue: !Sub "eksctl-biomage-${Environment}-cluster::ClusterSecurityGroupId"
+      RotationRules:
+        Duration: 2h
```

Comment:
This rotates every 2 hours? Is it necessary to do it so frequently I looked at some docs on rotation and usually they do it every 30 days or something

Reply:
No, that is the rotation time window. That means the key will be rotated within 2 hours of the schedule.

Reply:
ah ok, then how frequent is the rotation?

Reply:
This is defined in `ScheduleExpression: 'cron(0 8 1 * ? *)`

Reply:
Nice, coukld you add a comment next to `ScheduleExpression` saying that the rotation is every 1 month?